### PR TITLE
Fix failed test cases after 2.23 release

### PR DESCRIPTION
**Pull request by bilalawan321**

### CODE REVIEW

**Comment by bilalawan321** [comment]
@farhan-arshad-dev can you please review this PR. ?

File: tests/android/pages/android_elements.py [hunk 1]
```diff
@@ -93,7 +97,7 @@
 register_error_alert_textview = 'android:id/message'
 register_error_alert_button = 'android:id/button1'
 register_validate_editfield_error_textview = 'org.edx.mobile:id/input_error'
-register_validate_spinner_error_textview = 'org.edx.mobile:id/input_spinner_error'
+register_validate_spinner_error_textview = 'org.edx.mobile:id/input_auto_complete_error'
```
Comment (farhan-arshad-dev) [inline tests/android/pages/android_elements.py:100]: `register_validate_auto_complete_error_textview` is more appropriate.
use the following statment to make the prevent from the `pylint` error.
```
register_validate_auto_complete_error_textview = 'org.edx.mobile:id/' \
                                           'input_auto_complete_error'
```

**Review by farhan-arshad-dev — CHANGES_REQUESTED** [review]

File: tests/android/pages/android_elements.py [hunk 2]
```diff
@@ -73,14 +73,18 @@
...
 register_input_instruction_all_textviews = 'org.edx.mobile:id/input_instructions'
 register_all_spinners = 'org.edx.mobile:id/input_spinner'
+register_all_auto_complete = 'org.edx.mobile:id/input_auto_complete'
 register_screen_tvs = 'android.widget.TextView'
 register_divider_textview = 'org.edx.mobile:id/divider_with_text_tv'
 register_facebook_textview = 'org.edx.mobile:id/fb_tv'
 register_google_textview = 'org.edx.mobile:id/google_tv'
 register_with_email_divider_textview = 'org.edx.mobile:id/divider_with_text_tv'
+# register_country_textview = 'org.edx.mobile:id/ll_register_auto_complete'
+register_country_textview = 'org.edx.mobile:id/input_auto_complete'
```
[the remark this answers is not in the document]
Comment (bilalawan321) [reply tests/android/pages/android_elements.py]: fixed

File: tests/android/pages/android_elements.py (hunk 1 above)
Comment (bilalawan321) [reply tests/android/pages/android_elements.py]: fixed

**Comment by bilalawan321** [comment]
@farhan-arshad-dev i have fixed your reviews, can you check now. ?

**Review by farhan-arshad-dev — APPROVED** [review]

**Pull request merged**